### PR TITLE
1038 Update Integrate Flexible Classroom changes.

### PR DESCRIPTION
**Pull request by nirm2009**
Integrate Flexible Classroom document updates for Android, iOS, Web, and Electron platforms.

### CODE REVIEW

**Review by atovpeko — CHANGES_REQUESTED** [review]
Please address a few minor language things.

File: shared/flexible-classroom/integrate-flexible-classroom/electron.mdx [hunk 1]
```diff
@@ -1,96 +1,95 @@
...
+      "electron_version": "12.0.0",
+      "prebuilt": true
+    },
+    "dependencies": {
+      ...
+      "agora-electron-sdk": "3.5.1-build.1116",
+    }
+    ```
 
 4. Import the Agora Classroom SDK in the HTML file and launch a classroom:
```
[the remark this answers is not in the document]
Comment (nirm2009) [reply shared/flexible-classroom/integrate-flexible-classroom/electron.mdx:59]: Hello @atovpeko,
I'm afraid it is impossible in MDX to place a variable within quotes. As a workaround, may I suggest we replace "3.5.1-build.1116" with <build version>?

File: shared/flexible-classroom/integrate-flexible-classroom/ios.mdx [hunk 2]
```diff
@@ -1,143 +1,146 @@
...
+            - Set `Key` as SWIFT_VERSION.
+            - Set `Value` as your required Swift version.
 
-- If your project is an OC project, you need to add User-Defined Setting in Build Settings:
-   - Set Key as SWIFT_VERSION.
-   - Set Value as the Swift version that you want.
+    - For Swift projects:
+        Use `import AgoraEduCorePuppet` to import `AgoraEduCore`.
 
+    - Replace the `AgoraEduCore` class with the `AgoraEduCorePuppet` class.
```
[the remark this answers is not in the document]
Comment (nirm2009) [reply shared/flexible-classroom/integrate-flexible-classroom/ios.mdx]: Hey @atovpeko, thanks for the suggestions! I changed the section to use a numbered list. The last bullet was mentioned as the last step for both Swift and Objective-C (before my changes on this PR), so if it's alright, I think we should keep it this way.

File: shared/flexible-classroom/integrate-flexible-classroom/electron.mdx (hunk 1 above)
[the remark this answers is not in the document]
Comment (atovpeko) [reply shared/flexible-classroom/integrate-flexible-classroom/electron.mdx:59]: Oh no, I meant for Classroom SDK, if there is one :-)

**Review by atovpeko — APPROVED** [review]

**Pull request merged**